SPECS/r8152-module-alt.spec

```diff
@@ -0,0 +1,66 @@
+%define module_dir updates
+
+Summary: Driver for Realtek r8152
+Name: r8152-module-alt
+Version: 2.19.2
+Release: 1%{?dist}
+License: GPL
+# Sources extracted from the Linux kernel %{version}
+Source: %{name}-%{version}.tar.gz
```

Comment:
the comment says that the source comes form the kernel but the version number suggests it comes from realtek.
@andrew64k could you tell us where the source comes from?

Reply:
I've found it at https://github.com/wget/realtek-r8152-linux/releases/tag/v2.19.2

Reply:
Correct, my mistake. It is direct Realtek source code, not upstream Linux kernel code.

It's not from that github repo, but directly from [Realtek web site](https://www.realtek.com/Download/List?cate_id=585).